specification/dimensions/dimensions.mdpp

```diff
@@ -17,6 +17,7 @@ dimensions to categorize, filter, and reveal details in your data when grouped w
 !INCLUDE "commitmentdiscountid.md",1
 !INCLUDE "commitmentdiscountname.md",1
 !INCLUDE "commitmentdiscountcategory.md",1
```

Comment:
Oh snap, can you also move this to be sorted alphabetically?

Reply:
How on earth did that get past my borderline OCD?!?!!? Reordered